### PR TITLE
support offline mode, support icons and fonts

### PR DESCRIPTION
**Pull request by aneeshrelan**
- [x] Moved all usage of `<Icon>{icon}</Icon>` to dedicated icons from material-ui. Except the usage of dynamic application icons
- [x] Included Robot font within app.css
- [x] Validated on an offline mode with no internet

Resolves [#863](https://github.com/Unleash/unleash/issues/863)

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -7,16 +7,7 @@
...
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <meta name="description" content="unleash" />
-
         <title>Unleash - Enterprise ready feature toggles</title>
-        <link
-            href="https://fonts.googleapis.com/icon?family=Material+Icons"
-            rel="stylesheet"
-        />
-        <link
-            href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700"
```
Comment (FredrikOseberg) [inline public/index.html:17]: We need to include the fonts for the different weight types as well (300, 400, 500, 700) and add the correct font weight mapping in the @font-face ruleset. Right our fonts have only the default regular weight regardless of the CSS settings.

File: src/app.css [hunk 2]
```diff
@@ -1,3 +1,7 @@
+@font-face {
+    font-family: 'Roboto';
+    src: url('./assets/fonts/Roboto-Regular.ttf');
+}
```
Comment (FredrikOseberg) [inline src/app.css:4]: The font weight should be included here, as well as the other fonts.

File: src/component/addons/AddonList/ConfiguredAddons/ConfiguredAddons.jsx [hunk 3]
```diff
@@ -62,11 +63,7 @@ const ConfiguredAddons = ({
                             }
                             onClick={() => toggleAddon(addon)}
                         >
-                            <Icon>
-                                {addon.enabled
-                                    ? 'visibility'
-                                    : 'visibility_off'}
-                            </Icon>
+                            {addon.enabled ? <Visibility /> : <VisibilityOff />}
```
Comment (FredrikOseberg) [inline src/component/addons/AddonList/ConfiguredAddons/ConfiguredAddons.jsx:66]:
```suggestion
                           <ConditionallyRender 
                                  condition={addon.enabled} 
                                  show={<Visibility />} 
                                  elseShow={<VisibilityOff />}
                            />
```

File: src/component/menu/drawer.jsx [hunk 4]
```diff
@@ -17,8 +17,8 @@ const filterByFlags = flags => r => {
     return true;
 };
 
-function getIcon(name) {
-    if (name === 'c_github') {
+function getIcon(IconComponent) {
```
Comment (FredrikOseberg) [inline src/component/menu/drawer.jsx:20]: The "library_books" icon is actually coming from our API, which will override the default setting you are setting in the reducer further down and render `<library_books>` into the DOM causing a warning. We need to add a check here for "library_books" like we are doing with the github icon.

File: src/store/ui-config/index.js [hunk 5]
```diff
@@ -21,7 +22,7 @@ const DEFAULT = new $Map({
     links: [
         {
             value: 'Documentation',
-            icon: 'library_books',
+            icon: LibraryBooks,
```
Comment (FredrikOseberg) [inline src/store/ui-config/index.js:25]: This is coming dynamically from the API and will override these defaults. See the comment above.

**Review by FredrikOseberg — CHANGES_REQUESTED** [review]
Fantastic start. I added a few comments that we need to resolve before merging, but this is really close. 

If you can open a PR from a branch in your repository it would be easier for me to check out locally.

**Review by aneeshrelan — COMMENTED** [review]
> Fantastic start. I added a few comments that we need to resolve before merging, but this is really close.
> 
> If you can open a PR from a branch in your repository it would be easier for me to check out locally.

Thanks for the initial feedback. I'll resolve in a couple of days, if that's okay and probably re-create the PR to change the source branch

File: public/index.html [hunk 6]
```diff
@@ -7,16 +7,7 @@
         <meta name="baseUriPath" content="::baseUriPath::" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <meta name="description" content="unleash" />
-
         <title>Unleash - Enterprise ready feature toggles</title>
-        <link
```
[the remark this answers is not in the document]
Comment (FredrikOseberg) [reply public/index.html:12]: Yes you're correct. We'll keep it in place for now.

**Pull request closed without merging**